src/Resources/contao/languages/en/tl_content.php

```diff
@@ -17,7 +17,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderButtons'] = ['Prev/next buttons','Shows the prev/next buttons.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPagination'] = ['Pagination','Shows a clickable pagination to navigate the slides.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPaginationType'] = ['Pagination type','The type of the pagination.'];
-$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Autoheight','When active, the height of the slider adapts to the height of the currently active element.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Auto-Height','When active, the height of the slider adapts to the height of the currently active element.'];
```

Comment:
Height should be in lower case here I think ;). Also I think you can omit the comma after "When active" (different comma rules for English).